pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -89,12 +89,8 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 		},
 	}
 	flags := cmd.Flags()
-
-	releaseVer, err := version.ParseGitVersion(version.Get().GitVersion)
-	if err != nil {
-		klog.Infof("No default release version found. build version: %s", version.Get().String())
-		releaseVer = &version.ReleaseVersion{} // initialize to avoid panic
-	}
+	// image registry
+	flags.StringVarP(&opts.ImageRegistry, "image-registry", "", "", "Image registry.")
```

Comment:
We could add more detail description for `image-registry`.

Reply:
```go
Image registry. Registry containing karmada all image
```
Whether such a description is readable ?

Reply:
https://github.com/kubernetes/kubernetes/blob/449c46258b0e36b80d590ffcc26bd43ecfb60132/cmd/kubeadm/app/cmd/options/generic.go#L64
```go
Choose a container registry to pull control plane images from
```

Reply:
It looks good.

cc @RainbowMango